src/Servers/IIS/AspNetCoreModuleV2/CommonLib/ServerErrorHandler.h

```diff
@@ -9,31 +9,40 @@
 class ServerErrorHandler : public REQUEST_HANDLER
 {
 public:
+    ServerErrorHandler(IHttpContext& pContext, USHORT statusCode, USHORT subStatusCode, std::string statusText, HRESULT hr, HINSTANCE module, bool disableStartupPage, BYTE* content, int length) noexcept
+        : ServerErrorHandler(pContext, statusCode, subStatusCode, statusText, hr, module, disableStartupPage, 0, content, length)
+    {
+    }
+
+    ServerErrorHandler(IHttpContext& pContext, USHORT statusCode, USHORT subStatusCode, std::string statusText, HRESULT hr, HINSTANCE module, bool disableStartupPage, int page) noexcept
```

Comment:
There are two scenarios here. One which loads from a resource file (page number passed in) and the other is passed in via the startup hook. I have a 3rd constructor which takes both.